syntax = "proto3";

package proto;

service ReplicationService {
  rpc SendReplication(Replica) returns (confirmationMessage);
  rpc getReplication(confirmationMessage) returns (Replica);
}

message confirmationMessage {
  string messageOfConfirmation = 1;
}

message Replica {
  bytes data = 1;
}
